sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -1089,12 +1089,11 @@
               - `CREATOR_SW_CFG_FLASH_INFO_BOOT_DATA_CFG` controls the scrambling, ecc, and he
                 settings for the boot partitions `kFlashCtrlInfoPageBootData0` and
                 `kFlashCtrlInfoPageBootData1`.
-            - Verify that `kFlashCtrlInfoPageCreatorSecret` is locked down.
             - Verify that flash_ctrl is initialized.
             '''
       tags: ["rom", "no_dv", "fpga", "silicon"]
       stage: V2
-      tests: ["rom_keymgr_init"]
+      tests: ["rom_e2e_flash_ctrl_init"]
```

Comment:
@timothytrippel we should leave this empty, otherwise it will show up in the dv dashboard, right?

Reply:
this will have no effect since any test with the `no_dv` tag will not end up in the dashboard; only if there are plans to add it to dv, then we should leave it blank until its added to DV.

Reply:
though since it has no impact, it would be cleaner / more consistent if this was empty